project_report/tex/requirements.tex

```diff
@@ -9,7 +9,9 @@ \subsubsection{Functional \& Non-Functional Requirements}
 \subsubsection{The Physical Setup (The Brewery Machine)}
 
 \subsubsection{The Simulator}
```

Comment:
Maybe write about how we are going to use the simulator throughout our project? 
Be specific on this one, as we need this section to be relevant to our report.